lib/api/core/models/repositories/roleRepository.js

```diff
@@ -8,212 +10,219 @@ var
 /**
  * @class RoleRepository
  * @extends Repository
- * @constructor
  */
-function RoleRepository() {
-  Repository.apply(this, arguments);
-  this.collection = 'roles';
-  this.ObjectConstructor = Role;
-  this.roles = {};
-}
+class RoleRepository extends Repository {
+  /**
+   * @constructor
+   * @param {Kuzzle} kuzzle
+   */
+  constructor (kuzzle) {
+    super(kuzzle);
+
+    this.collection = 'roles';
+    this.ObjectConstructor = Role;
+    this.roles = {};
+  }
+
+  init () {
+    super.init({});
+  }
 
-RoleRepository.prototype = Object.create(Repository.prototype);
-RoleRepository.prototype.constructor = RoleRepository;
+  /**
+   * From a list of role ids, retrieves the matching Role objects.
+   *
+   * @param {Array} roleKeys The role ids to load
+   * @returns {Promise} Resolves to an array containing the matching found Role objects.
+   */
+  loadRoles (roleKeys) {
+    const
+      keys = [],
+      buffer = {},
+      result = [];
+    let role;
```

Comment:
nitpicking: can be scoped inside the ` else if (this.kuzzle.config.security.standard.roles[roleKey])` block